typescript/api/controllers/UserController.ts

```diff
@@ -285,7 +285,7 @@ export module Controllers {
           }
 
           let oidcConfig = _.get(sails.config, 'auth.default.oidc');
-          let errorMessage = _.get(err, 'message');
+          let errorMessage = _.get(err, 'message', err?.toString() ?? '');
```

Comment:
I added `err?.toString()` because sometime the `err` was an `Error`, and sometimes it was a string.